nce/NCE_MP_Pytorch/evaluators/qa_evaluator.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
```

Comment:
What's the difference between this evaluator and the one I have for MP-CNN? They are both trying to calculate map and mrr right?

Reply:
There is a minor difference in the model prediction. Moreover, no loss number is output during this step in NCE version.

Reply:
I would suggest making a general evaluator that both your code and my code can use. If your's is more general then I can use your's. Also it doesn't hurt to return loss if you just don't use it?

Reply:
I see. qa_evaluator.py can be combined with yours. I will do it.